### PR TITLE
fixed issue 1561

### PR DESCRIPTION
**Pull request by PeterNgTr**
**What are you trying to achieve?**
I want to test if the exact text "Authentication failed." appears in the p subelement of

```
<div class="ui error message">
  <div class="content">
    <p></p>
  </div>
</div>
```

**Let's say now there is no value in p.**
`So I.seeTextEquals("Authentication failed.", ".error.message"); should fail.`

### CODE REVIEW

File: test/helper/WebDriver_test.js [hunk 1]
```diff
@@ -198,6 +198,13 @@ describe('WebDriver', function () {
         // e.should.be.instanceOf(AssertionFailedError);
         // e.inspect().should.include("expected element h1 'Welcome to test app' to equal 'Welcome to test app!'");
       }));
+
+    it.only('should check text is not equal to empty string of element text', () => wd.amOnPage('https://codecept.discourse.group/')
```
Comment (DavertMik) [inline test/helper/WebDriver_test.js]: ONLY!

Comment (DavertMik) [reply test/helper/WebDriver_test.js]: Your linter did a lot of fixes but it bypasses this pretty important one. 
I'm using "eslint-plugin-mocha" for this kind of checks. You can try it too )

Comment (PeterNgTr) [reply test/helper/WebDriver_test.js]: I'm using the `npm run lint-fix` in the repo :D

**Comment by DavertMik** [comment]
Thank you!

**Pull request merged**